core/arch/x86/decode_table.c

```diff
@@ -5092,7 +5092,7 @@ const instr_info_t prefix_extensions[][12] = {
     {INVALID,  0xf338f018, "(bad)",   xx, xx, xx, xx, xx, no, x, NA},
     /* really this is regular data-size prefix */
     {OP_movbe, 0x6638f018, "movbe", Gw, xx, Mw, xx, xx, mrm, x, tpe[139][2]},
-    {OP_crc32, 0xf238f018, "crc32", Gv, xx, Eb, Gv, xx, mrm, x, END_LIST},
+    {OP_crc32, 0xf238f018, "crc32", Gd_q, xx, Eb, Gd_q, xx, mrm, x, END_LIST},
```

Comment:
Looking at the manual, yes for the destination. But the source seems to be always 1-byte for the F0 version, and 2/4/8 bytes for the F1. Why Gd_q for the source?

Reply:
I think you're mixing up the separate source Eb from the destination-is-also-a-source.

Reply:
I was! Ok, looks good.